src/main/java/com/onlinebookstore/onlinebookstore/model/CartItem.java

```diff
@@ -0,0 +1,37 @@
+package com.onlinebookstore.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Setter
+@Getter

```

Comment:
let's also not forget abut equals, hashcode and toString, jsut exclude lazy fields form them